app/main/utils.py

```diff
@@ -74,4 +74,29 @@ def create_meeting_notes(meeting_date,
                    new_value=meeting_notes.val_for_events)
     create_object(event)
 
-    return meeting_notes.id
\ No newline at end of file
+    return meeting_notes.id
+
+
+def create_news(author,
+                title,
+                content,
+                tags):
+    """
+    Util function for creating a News object. Function will take parameters passed in from the form
```

Comment:
I would suggest (maybe as a todo) that we list types for the inputs. 

```suggestion
Util function for creating a News object. Function will take parameters passed in from the form and create a News along with the event object.

Args:
    author (String): The name of the Author
    title (String): Title of the news post
    content (String): Formatted  content of the post.
    tags (List): List of tag IDs for the post

Returns:
    post_id (Integer): ID of the newly created post.
```